lib/apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -37,12 +37,12 @@ type handler interface {
 	Configure(api *operations.PortLayerAPI, handlerCtx *handlers.HandlerContext)
 }
 
-var portlayerhandlers = []handler{
-	&handlers.StorageHandlersImpl{},
-	&handlers.MiscHandlersImpl{},
-	&handlers.ScopesHandlersImpl{},
-	&handlers.AttachHandlersImpl{},
-	&handlers.ContainersHandlersImpl{},
+var portlayerhandlers = map[string]handler{
```

Comment:
Does this compile?  I think this refers to previous revisions.  You may want to revert this back to a list.